.github/workflows/ci.yaml

```diff
@@ -100,7 +100,12 @@ jobs:
       - run: python -c 'import os; print(os.getenv("GOOGLE_APPLICATION_CREDENTIALS_JSON", "").strip())' > ${{ env.GOOGLE_APPLICATION_CREDENTIALS }}
       - run: sqlite3 /tmp/sqlite_default.db "VACUUM;"
       - run: pip3 install nox
-      - run: nox -s test -- --splits 8 --group ${{ matrix.group }}
+      - run: nox -s test -- --splits 8 ${{ matrix.group }} --cov=./ --cov-report=xml
```

Comment:
Please, add `--cov-branch` as well (if possible both in the CI and in the dev documentation)
https://pytest-cov.readthedocs.io/en/latest/config.html
https://coverage.readthedocs.io/en/6.3.1/branch.html